Library/Formula/letsencrypt.rb

```diff
@@ -0,0 +1,37 @@
+class Letsencrypt < Formula
+  desc "The Let's Encrypt Client is a tool to automatically receive and install X.509 certificates"
```

Comment:
You can shorten this: `"tool to automatically receive and install X.509 certificates"`.